tests/sorcha/test_PPLinkingFilter.py

```diff
@@ -1,34 +1,131 @@
 import pandas as pd
 import numpy as np
-from numpy.testing import assert_equal
 
 from sorcha.utilities.dataUtilitiesForTests import get_test_filepath
 
 
 def test_PPLinkingFilter():
```

Comment:
These tests look really good to me! Good sanity checks of some of the parameters in difi.